server/src/main/java/org/opensearch/cluster/decommission/DecommissionService.java

```diff
@@ -481,4 +482,51 @@ public void onFailure(Exception e) {
             }
         };
     }
+
+    public void deleteDecommissionAttribute(final ActionListener<AcknowledgedResponse> listener) {
```

Comment:
Why is this `AcknowledgedResponse`? Shouldn't it be `DeleteDecommissionResponse`?

Reply:
If it is because of pending API PR.. please add a TODO here

Reply:
Link: https://github.com/opensearch-project/OpenSearch/pull/4320#discussion_r984808580